Gemfile

```diff
@@ -1,6 +1,4 @@
 source 'https://rubygems.org'
-git_source(:github) { |repo| "git@github.com:#{repo}.git" }
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
```

Comment:
I ended up removing `gem 'workarea'` because this should be testing against a released version of the platform, but decided to leave this in so the next person who tries to add a gem doesn't run into SSH permissions issues trying to access a private repo, which the CI server will never have access to.